### PR TITLE
更新checkin脚本；删除一部分空folder

### PR DESCRIPTION
**Pull request by sainas**
1. 修改了checkin脚本，本地测试通过。还设置了北京时区加了一行log
2. reader里修改了`danie10ne` 的名字，因为看ta的笔记是 `daniel.md`
3. 删除了2023-05-22之前和10月之后的folder。不知道为什么感觉这个repo的`git status`和一些git操作都有点慢，猜想是不是folder太多的原因。感觉可以等三个月如果有需要再generate。

### CODE REVIEW

File: checkin.py [hunk 1]
```diff
@@ -10,9 +11,15 @@
     users = [line.strip() for line in f.readlines()]
 
 # 获取当前日期和前两天日期
-today = datetime.date.today()
+beijing_timezone = pytz.timezone('Asia/Shanghai')
```
Comment (wfnuser) [inline checkin.py:14]: Can you provide a direct result you get from your console for me. It will be easier to know the effect.

**Comment by wfnuser** [comment]
It's cool to delete the files before 2023-05-22. But it's meaningless to delete the future ones, since we will be there soon.
And if you thought git command don't run fast enough, I think you'd better find other reason for it. It shouldn't be slow for this large files.

And please rebase first. I already change the line `folder_path = folder.strftime('notes/%Y/%m/%d')`.

Thanks for contribute.

File: checkin.py (hunk 1 above)
Comment (sainas) [reply checkin.py:14]:
```shell
➜   ./checkin.py
今天是：2023-05-26, 检查过去两天打卡情况: 2023-05-25 2023-05-24
当前6位同学参与打卡，1位老板缺卡，缺卡率：16.67%。请发2元红包5份，再发一个2元红包给我。谢谢老板！
```

**Comment by sainas** [comment]
> It's cool to delete the files before 2023-05-22. But it's meaningless to delete the future ones, since we will be there soon. And if you thought git command don't run fast enough, I think you'd better find other reason for it. It shouldn't be slow for this large files.
> 
> And please rebase first. I already change the line `folder_path = folder.strftime('notes/%Y/%m/%d')`.
> 
> Thanks for contribute.

Changed to only delete old empty folders, since I can't prove they are the sole reason for the slowness right now.
Rebased.

File: checkin.py (hunk 1 above)
Comment (wfnuser) [reply checkin.py:16]: Cool.

File: checkin.py [hunk 2]
```diff
@@ -1,5 +1,8 @@
+#!/usr/bin/env python3
+
 import os
 import datetime
+import pytz
```
Comment (wfnuser) [inline checkin.py:5]: @sainas  Hey. I have another concern about it. Do we need to install pytz. That will be inconvenient.
Can you used some library out of the box, like ZoneInfo.

**Review by wfnuser — CHANGES_REQUESTED** [review]

File: checkin.py (hunk 2 above)
Comment (wfnuser) [reply checkin.py]: Cool.

**Pull request merged**